documentation/blog/2019-04-20-write-less-code.md

```diff
@@ -40,7 +40,7 @@ Reducing the amount of code you have to write is an explicit goal of Svelte. To
 <div class="max">
 	<iframe
 		title="Simple component example"
-		src="/repl/embed?example=blog-write-less-code"
+		src="/repl/blog-write-less-code/embed?version=4.2.15"
```

Comment:
I'm torn about whether to include a `?version` in this URL and about how specific to make it. When this blog post came out, it was for Svelte 3.0. We're now on v4, but someday soon-ish we'll be on 5.0.

We could omit the `?version` entirely, or we could make it `?version=3` which will match the latest v3 version, or we could make it `?version=4` which will match the latest v4 version. I don't think, though, that we want to pin a specific version of Svelte.

Reply:
I think it should be `?version=3` to keep the repl in the same context as when it was written. 

This logic can be applied to all blog posts.

Reply:
```suggestion
		src="/repl/blog-write-less-code/embed?version=3"
```